base/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM golang:1.12.9-stretch AS builder
+
+MAINTAINER free5GC <support@free5gc.org>
+
+ENV GO111MODULE=off
+
+RUN apt-get update
+RUN apt-get -y install gcc cmake autoconf libtool pkg-config libmnl-dev libyaml-dev
+RUN apt-get clean
+
+# Get Free5GC
+RUN cd $GOPATH/src \
+    && git clone --recursive -b v3.0.2 -j 33 https://github.com/free5gc/free5gc.git \
+    && cd free5gc \
+    && git submodule update --init -j 33 \
```

Comment:
I think there is no need to have `submodule update` after using `git cloen --recursive`?

Reply:
Right. Sorry for that.